include/cxxopts.hpp

```diff
@@ -1456,6 +1456,7 @@ namespace cxxopts
             stringAppend(result, "\n");
             stringAppend(result, start, ' ');
             startLine = lastSpace + 1;
+            lastSpace = startLine;
```

Comment:
what's this change for?

Reply:
That's a commit from #167. My bad. Fixed now...